app/api/leaderboard/functions.ts

```diff
@@ -70,62 +70,57 @@ export default async function fetchGitHubReleases(
   if (!accessToken) {
     return [];
   }
-
-  const response = await fetch("https://api.github.com/graphql", {
-    method: "POST",
-    headers: {
-      Authorization: `Bearer ${accessToken}`,
-      "Content-Type": "application/json",
-    },
-    body: JSON.stringify({
-      query: `{
-        organization(login: "${env.NEXT_PUBLIC_GITHUB_ORG}") {
-          repositories(first: 100) {
-            nodes {
-              name
-              releases(first: 10, orderBy: {field: CREATED_AT, direction: DESC}) {
-                nodes {
-                  name
-                  createdAt
-                  description
-                  url
-                  author {
-                    login
-                    avatarUrl
-                  }
-                  mentions (first: 10) {
-                    nodes {
-                      login 
+  try {
```

Comment:
let's not wrap it in a try-catch. Let the build fail if there is an error, since this would be taken care during pre-rendering.